packages/dapp-connector/src/WalletApi/types.ts

```diff
@@ -206,5 +209,11 @@ export interface Cip95WalletApi {
 }
 
 export type WalletApi = Cip30WalletApi & Cip95WalletApi;
-
 export type WalletMethod = keyof WalletApi;
+
+export type cipName = `cip${number}`;
+export interface CipExtensionApis {
+  [cip95: cipName]: Cip95WalletApi;
+}
+
+export type NamespacedWalletApi = Cip30WalletApi & Partial<CipExtensionApis>;
```

Comment:
I wouldn't call it `NamespacedWalletApi` because you can create extensions without namespacing. I think the original `WalletApi` is fine, and we could have like
```
type Cip95Api = { ... };
type Cip95WalletApi = { cip95: Cip95Api; }
```

Reply:
Was this addressed by https://github.com/input-output-hk/cardano-js-sdk/pull/930/commits/6e19631bcc18d700dd8023be676a06cbd3edecb1 ?